src/Microsoft.AspNetCore.Sockets.Client.Http/LongPollingTransport.cs

```diff
@@ -23,7 +23,7 @@ public partial class LongPollingTransport : ITransport
         private IDuplexPipe _application;
         private Task _sender;
         private Task _poller;
-
+        private HttpResponseMessage _response;
```

Comment:
Why is this being stored?

Reply:
Because we needed to dispose the response in StopAsync. This is irrelevant now though as per Andrew's suggestion above